src/index.html

```diff
@@ -1,16 +1,50 @@
 <!doctype html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
-    <meta
-      name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
-    >
+    <meta name="viewport"
+    content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action=""
+    data-qa="big"
+    >
+      <label for="search__big">
+        <img
+        class="search__big__img"
+        src="images/Search.svg"
+        alt="Search">
+      </label>
+      <input
+      id="search__big"
+      type="text"
+      data-qa="keypress"
+      placeholder="Try &quot;Los Angeles&quot;"
+      class="search__input search__big__input"
+      >
+    </form>
+    <form
+    action=""
+    data-qa="small"
+    >
+      <label for="search-small">
+        <img
+        class="search__small__img"
+        src="images/Search.svg"
+        alt="Search"
+        >
+      </label>
+      <input
+      id="search-small"
+      type="text"
+      placeholder="Try &quot;Los Angeles&quot;"
+      class="search__small__input search__input">
+    </form>
   </body>
-</html>
+</html>
```

Comment:
```suggestion
</html>

```
add an empty line of the end of file